src/components/EditableTextarea/styles/EditableTextarea.css.ts

```diff
@@ -121,11 +115,6 @@ export const EditableTextareaUI = styled('div')`
     height: auto;
     border: 0;
     margin: 0;
-    ${/* 
-      I have no idea why Chrome and safari are adding 3px to the height of the parent of the textarea,
-      so I'm adding those 3px to firefox for parity  ¯\_(ツ)_/¯
-    */
-    isFirefox() && 'margin-bottom: 3px;'}
```

Comment:
Is removing this having some undesirable effect that I'm not noticing? I needed to take it away in order for the border and fader to play nicely together on Firefox.

Reply:
I added this because chrome and webkit add some pixels to the container of the textarea, making firefox look a bit smaller, it was so that they look almost exactly the same, but I didn't notice the issue of the mismatched borders you're solving here.

If we just remove this, I don't see the issue anymore (maybe I'm missing something), so maybe the rest of the changes are not needed? Having all browsers to look exactly the same to the pixel is often futile, I thought this was an easy win, but it clearly wasn't!

🙈